### ISSUE TITLE
Low performance alerting

### ISSUE THREAD

**Issue by ainemckay** [MHRA/products#409]
### User want
As a member of the Products support team I would like the team to be alerted when the site isn't performing, so that we can react quickly to fix any issues.

**Customer acceptance criteria** 
1. The business receives alerts in an agreed timescale and format

**Technical acceptance criteria**
1. We need a distribution list (Andy Snow to create) on AD
3. An alert should be sent if the API is not accessible (DDOS and error handling)
4. All alerts are documented in the repository

**Data acceptance criteria**
Everything should be logged on Azure Monitoring

**Size**
L

**Value**

**Effort**

### Exit Criteria met 
- [x] Backlog
- [x] Discovery
- [x] DUXD
- [ ] Development 
- [ ] Quality Assurance 
- [ ] Release and Validate

**Comment by m-doughty** [comment]
Further to discussions with Andy Snow, we're going to implement some basic alerting for catastrophic failure or large cost changes. This is because the front end doesn't currently use the API in production, so we have no normative load data from which to create assumptions about what 'high' usage is.

The strategy beyond this will be to monitor production after #681 & related tickets are delivered to production so that we can establish 'normal' usage patterns and any further necessary alerts can be implemented..

**Comment by majikandy** [comment]
Decision with @ainemckay to move cost alerting out of this card and into its own card. So that this card can move forward as it can be tested and delivered independently
